config/initializers/sidekiq.rb

```diff
@@ -20,4 +20,13 @@
   }
 })
 
+Sidekiq::Throttled::Registry.add(:tulsa_city, **{
+  threshold: {
+    :limit => ->(*_) {
+      Time.now.utc.between?(OKLAHOMA_DAY_START, OKLAHOMA_DAY_END) ? 60.to_i : 1_000
```

Comment:
60.to_i - pull this from an env variable
ENV.fetch('TULSA_CITY_THROTTLE', 60).to_i